android/src/main/java/com/decodedhealth/flutter_zoom_plugin/ZoomView.java

```diff
@@ -141,6 +147,42 @@ private void joinMeeting(MethodCall methodCall, MethodChannel.Result result) {
         result.success(true);
     }
 
+    private void startMeeting(MethodCall methodCall, MethodChannel.Result result) {
+
+        Map<String, String> options = methodCall.arguments();
+
+        ZoomSDK zoomSDK = ZoomSDK.getInstance();
+
+        if(!zoomSDK.isInitialized()) {
+            System.out.println("Not initialized!!!!!!");
+            result.success(false);
+            return;
+        }
+
+        final MeetingService meetingService = zoomSDK.getMeetingService();
+
+        StartMeetingOptions opts = new StartMeetingOptions();
+        opts.no_invite = parseBoolean(options, "disableInvite", false);
+        opts.no_share = parseBoolean(options, "disableShare", false);
+        opts.no_driving_mode = parseBoolean(options, "disableDrive", false);
+        opts.no_dial_in_via_phone = parseBoolean(options, "disableDialIn", false);
+        opts.no_disconnect_audio = parseBoolean(options, "noDisconnectAudio", false);
+        opts.no_audio = parseBoolean(options, "noAudio", false);
+
+        StartMeetingParamsWithoutLogin params = new StartMeetingParamsWithoutLogin();
+
+		params.userId = options.get("userId");
+        params.displayName = options.get("displayName");
+        params.meetingNo = options.get("meetingId");
+		params.userType = MeetingService.USER_TYPE_API_USER;
+		params.zoomToken = options.get("zoomToken");
```

Comment:
Since the user token and Zoom access token are necessary for the non-login user to use Zoom, I would suggest adding a function to get these token for the user in the plugin, instead of leaving it out.